features/storage/csi_resize.feature

```diff
@@ -0,0 +1,45 @@
+Feature: CSI Resizing related feature
+  # @author chaoyang@redhat.com
+  @admin
+  Scenario Outline: Resize online volume from 1Gi to 2Gi
```

Comment:
Do you mind rephrase this scenario? What does `online volume` mean?
For example:`Increase the volume size`

Reply:
/lgtm

Reply:
`online`  means that volume is in-used by a pod. We don't need to restart the pod when doing the volume expansion.